libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -156,6 +156,8 @@ Opcode constOpcodeFor(ValueType _type)
 }
 
 static map<string, uint8_t> const builtins = {
+	{"i32.select", 0x1b},
+	{"i64.select", 0x1b},
```

Comment:
Does this list not say anywhere how many ins/outs a instruction has?

Reply:
Apparently not. Only the dialect has it.